<?xml version="1.0" encoding="UTF-8"?>
<testsuite xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="https://maven.apache.org/surefire/maven-surefire-plugin/xsd/surefire-test-report.xsd" name="org.springframework.data.redis.connection.lettuce.LettuceReactiveGeoCommandsTests" time="0" tests="28" errors="28" skipped="0" failures="0">
  <properties>
    <property name="java.runtime.name" value="OpenJDK Runtime Environment"/>
    <property name="sun.boot.library.path" value="/usr/lib/jvm/java-8-openjdk-amd64/jre/lib/amd64"/>
    <property name="java.vm.version" value="25.121-b13"/>
    <property name="java.vm.vendor" value="Oracle Corporation"/>
    <property name="maven.multiModuleProjectDirectory" value="/root/workspace/spring-projects/spring-data-redis/388703425"/>
    <property name="java.vendor.url" value="http://java.oracle.com/"/>
    <property name="path.separator" value=":"/>
    <property name="guice.disable.misplaced.annotation.check" value="true"/>
    <property name="skip.gulp" value="true"/>
    <property name="java.vm.name" value="OpenJDK 64-Bit Server VM"/>
    <property name="file.encoding.pkg" value="sun.io"/>
    <property name="sun.java.launcher" value="SUN_STANDARD"/>
    <property name="sun.os.patch.level" value="unknown"/>
    <property name="findbugs.skip" value="true"/>
    <property name="java.vm.specification.name" value="Java Virtual Machine Specification"/>
    <property name="user.dir" value="/root/workspace/spring-projects/spring-data-redis/388703425"/>
    <property name="checkstyle.skip" value="true"/>
    <property name="java.runtime.version" value="1.8.0_121-8u121-b13-1~bpo8+1-b13"/>
    <property name="java.awt.graphicsenv" value="sun.awt.X11GraphicsEnvironment"/>
    <property name="skip.npm" value="true"/>
    <property name="java.endorsed.dirs" value="/usr/lib/jvm/java-8-openjdk-amd64/jre/lib/endorsed"/>
    <property name="os.arch" value="amd64"/>
    <property name="java.io.tmpdir" value="/tmp"/>
    <property name="gpg.skip" value="true"/>
    <property name="line.separator" value="&#10;"/>
    <property name="java.vm.specification.vendor" value="Oracle Corporation"/>
    <property name="cobertura.skip" value="true"/>
    <property name="os.name" value="Linux"/>
    <property name="classworlds.conf" value="/usr/share/maven/bin/m2.conf"/>
    <property name="sun.jnu.encoding" value="UTF-8"/>
    <property name="java.library.path" value="/usr/java/packages/lib/amd64:/usr/lib/x86_64-linux-gnu/jni:/lib/x86_64-linux-gnu:/usr/lib/x86_64-linux-gnu:/usr/lib/jni:/lib:/usr/lib"/>
    <property name="license.skip" value="true"/>
    <property name="java.specification.name" value="Java Platform API Specification"/>
    <property name="java.class.version" value="52.0"/>
    <property name="sun.management.compiler" value="HotSpot 64-Bit Tiered Compilers"/>
    <property name="skip.bower" value="true"/>
    <property name="maven.repo.local" value="/root/./workspace/spring-projects/spring-data-redis/388703425/.m2"/>
    <property name="os.version" value="3.10.0-862.3.2.el7.x86_64"/>
    <property name="user.home" value="/root"/>
    <property name="user.timezone" value="Europe/Paris"/>
    <property name="java.awt.printerjob" value="sun.print.PSPrinterJob"/>
    <property name="java.specification.version" value="1.8"/>
    <property name="file.encoding" value="UTF-8"/>
    <property name="user.name" value="root"/>
    <property name="java.class.path" value="/usr/share/maven/boot/plexus-classworlds-2.5.2.jar"/>
    <property name="skipITs" value="true"/>
    <property name="java.vm.specification.version" value="1.8"/>
    <property name="sun.arch.data.model" value="64"/>
    <property name="java.home" value="/usr/lib/jvm/java-8-openjdk-amd64/jre"/>
    <property name="sun.java.command" value="org.codehaus.plexus.classworlds.launcher.Launcher -B -D enforcer.skip=true -D dependency-check.skip=true -D skipITs=true -D cobertura.skip=true -D maven.repo.local=/root/./workspace/spring-projects/spring-data-redis/388703425/.m2 -D rat.skip=true -D findbugs.skip=true -D skip.bower=true -D gpg.skip=true -D checkstyle.skip=true -D skip.gulp=true -D license.skip=true -D skip.npm=true test"/>
    <property name="java.specification.vendor" value="Oracle Corporation"/>
    <property name="user.language" value="en"/>
    <property name="awt.toolkit" value="sun.awt.X11.XToolkit"/>
    <property name="dependency-check.skip" value="true"/>
    <property name="java.vm.info" value="mixed mode"/>
    <property name="java.version" value="1.8.0_121"/>
    <property name="java.ext.dirs" value="/usr/lib/jvm/java-8-openjdk-amd64/jre/lib/ext:/usr/java/packages/lib/ext"/>
    <property name="securerandom.source" value="file:/dev/./urandom"/>
    <property name="sun.boot.class.path" value="/usr/lib/jvm/java-8-openjdk-amd64/jre/lib/resources.jar:/usr/lib/jvm/java-8-openjdk-amd64/jre/lib/rt.jar:/usr/lib/jvm/java-8-openjdk-amd64/jre/lib/sunrsasign.jar:/usr/lib/jvm/java-8-openjdk-amd64/jre/lib/jsse.jar:/usr/lib/jvm/java-8-openjdk-amd64/jre/lib/jce.jar:/usr/lib/jvm/java-8-openjdk-amd64/jre/lib/charsets.jar:/usr/lib/jvm/java-8-openjdk-amd64/jre/lib/jfr.jar:/usr/lib/jvm/java-8-openjdk-amd64/jre/classes"/>
    <property name="java.vendor" value="Oracle Corporation"/>
    <property name="enforcer.skip" value="true"/>
    <property name="rat.skip" value="true"/>
    <property name="maven.home" value="/usr/share/maven"/>
    <property name="file.separator" value="/"/>
    <property name="java.vendor.url.bug" value="http://bugreport.sun.com/bugreport/"/>
    <property name="sun.cpu.endian" value="little"/>
    <property name="sun.io.unicode.encoding" value="UnicodeLittle"/>
    <property name="sun.cpu.isalist" value=""/>
  </properties>
  <testcase name="geoAddShouldAddMultipleGeoLocationsCorrectly[org.springframework.data.redis.connection.lettuce.StandaloneConnectionProvider@18c8a027]" classname="org.springframework.data.redis.connection.lettuce.LettuceReactiveGeoCommandsTests" time="0">
    <error message="Unable to connect to 127.0.0.1:6379" type="io.lettuce.core.RedisConnectionException">io.lettuce.core.RedisConnectionException: Unable to connect to 127.0.0.1:6379
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: syscall:getsockopt(..) failed: Connection refused: /127.0.0.1:6379
Caused by: io.netty.channel.unix.Errors$NativeConnectException: syscall:getsockopt(..) failed: Connection refused
</error>
    <system-out><![CDATA[2018-06-06 13:19:36,121  WARN .cluster.topology.ClusterTopologyRefresh: 151 - Unable to connect to 127.0.0.1:7379
java.util.concurrent.CompletionException: io.netty.channel.AbstractChannel$AnnotatedConnectException: syscall:getsockopt(..) failed: Connection refused: /127.0.0.1:7379
	at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:292)
	at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:308)
	at java.util.concurrent.CompletableFuture.uniApply(CompletableFuture.java:593)
	at java.util.concurrent.CompletableFuture.uniApplyStage(CompletableFuture.java:614)
	at java.util.concurrent.CompletableFuture.thenApply(CompletableFuture.java:1983)
	at io.lettuce.core.AbstractRedisClient.initializeChannelAsync(AbstractRedisClient.java:297)
	at io.lettuce.core.cluster.RedisClusterClient.connectStatefulAsync(RedisClusterClient.java:712)
	at io.lettuce.core.cluster.RedisClusterClient.connectToNodeAsync(RedisClusterClient.java:500)
	at io.lettuce.core.cluster.RedisClusterClient$NodeConnectionFactoryImpl.connectToNodeAsync(RedisClusterClient.java:1149)
	at io.lettuce.core.cluster.topology.ClusterTopologyRefresh.getConnections(ClusterTopologyRefresh.java:213)
	at io.lettuce.core.cluster.topology.ClusterTopologyRefresh.loadViews(ClusterTopologyRefresh.java:69)
	at io.lettuce.core.cluster.RedisClusterClient.loadPartitions(RedisClusterClient.java:846)
	at io.lettuce.core.cluster.RedisClusterClient.initializePartitions(RedisClusterClient.java:820)
	at io.lettuce.core.cluster.RedisClusterClient.connect(RedisClusterClient.java:346)
	at io.lettuce.core.cluster.RedisClusterClient.connect(RedisClusterClient.java:321)
	at org.springframework.data.redis.test.util.LettuceRedisClusterClientProvider.test(LettuceRedisClusterClientProvider.java:74)
	at org.springframework.data.redis.connection.lettuce.LettuceReactiveCommandsTestsBase.parameters(LettuceReactiveCommandsTestsBase.java:107)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.runners.Parameterized.allParameters(Parameterized.java:280)
	at org.junit.runners.Parameterized.<init>(Parameterized.java:248)
	at sun.reflect.GeneratedConstructorAccessor42.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at org.junit.internal.builders.AnnotatedBuilder.buildRunner(AnnotatedBuilder.java:104)
	at org.junit.internal.builders.AnnotatedBuilder.runnerForClass(AnnotatedBuilder.java:86)
	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)
	at org.junit.internal.builders.AllDefaultPossibilitiesBuilder.runnerForClass(AllDefaultPossibilitiesBuilder.java:26)
	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)
	at org.junit.internal.requests.ClassRequest.getRunner(ClassRequest.java:33)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:364)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:274)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:161)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: syscall:getsockopt(..) failed: Connection refused: /127.0.0.1:7379
	at io.netty.channel.unix.Socket.finishConnect(..)(Unknown Source)
Caused by: io.netty.channel.unix.Errors$NativeConnectException: syscall:getsockopt(..) failed: Connection refused
	... 1 common frames omitted
]]></system-out>
  </testcase>
  <testcase name="geoAddShouldAddMultipleGeoLocationsCorrectly[org.springframework.data.redis.connection.lettuce.StandaloneConnectionProvider@18c8a027]" classname="org.springframework.data.redis.connection.lettuce.LettuceReactiveGeoCommandsTests" time="0">
    <error message="Unable to connect to 127.0.0.1:6379" type="io.lettuce.core.RedisConnectionException">io.lettuce.core.RedisConnectionException: Unable to connect to 127.0.0.1:6379
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: syscall:getsockopt(..) failed: Connection refused: /127.0.0.1:6379
Caused by: io.netty.channel.unix.Errors$NativeConnectException: syscall:getsockopt(..) failed: Connection refused
</error>
  </testcase>
  <testcase name="geoPos[org.springframework.data.redis.connection.lettuce.StandaloneConnectionProvider@18c8a027]" classname="org.springframework.data.redis.connection.lettuce.LettuceReactiveGeoCommandsTests" time="0">
    <error message="Unable to connect to 127.0.0.1:6379" type="io.lettuce.core.RedisConnectionException">io.lettuce.core.RedisConnectionException: Unable to connect to 127.0.0.1:6379
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: syscall:getsockopt(..) failed: Connection refused: /127.0.0.1:6379
Caused by: io.netty.channel.unix.Errors$NativeConnectException: syscall:getsockopt(..) failed: Connection refused
</error>
  </testcase>
  <testcase name="geoPos[org.springframework.data.redis.connection.lettuce.StandaloneConnectionProvider@18c8a027]" classname="org.springframework.data.redis.connection.lettuce.LettuceReactiveGeoCommandsTests" time="0">
    <error message="Unable to connect to 127.0.0.1:6379" type="io.lettuce.core.RedisConnectionException">io.lettuce.core.RedisConnectionException: Unable to connect to 127.0.0.1:6379
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: syscall:getsockopt(..) failed: Connection refused: /127.0.0.1:6379
Caused by: io.netty.channel.unix.Errors$NativeConnectException: syscall:getsockopt(..) failed: Connection refused
</error>
  </testcase>
  <testcase name="geoDistShouldReturnDistanceInDesiredMetric[org.springframework.data.redis.connection.lettuce.StandaloneConnectionProvider@18c8a027]" classname="org.springframework.data.redis.connection.lettuce.LettuceReactiveGeoCommandsTests" time="0">
    <error message="Unable to connect to 127.0.0.1:6379" type="io.lettuce.core.RedisConnectionException">io.lettuce.core.RedisConnectionException: Unable to connect to 127.0.0.1:6379
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: syscall:getsockopt(..) failed: Connection refused: /127.0.0.1:6379
Caused by: io.netty.channel.unix.Errors$NativeConnectException: syscall:getsockopt(..) failed: Connection refused
</error>
  </testcase>
  <testcase name="geoDistShouldReturnDistanceInDesiredMetric[org.springframework.data.redis.connection.lettuce.StandaloneConnectionProvider@18c8a027]" classname="org.springframework.data.redis.connection.lettuce.LettuceReactiveGeoCommandsTests" time="0">
    <error message="Unable to connect to 127.0.0.1:6379" type="io.lettuce.core.RedisConnectionException">io.lettuce.core.RedisConnectionException: Unable to connect to 127.0.0.1:6379
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: syscall:getsockopt(..) failed: Connection refused: /127.0.0.1:6379
Caused by: io.netty.channel.unix.Errors$NativeConnectException: syscall:getsockopt(..) failed: Connection refused
</error>
  </testcase>
  <testcase name="geoHashNotExisting[org.springframework.data.redis.connection.lettuce.StandaloneConnectionProvider@18c8a027]" classname="org.springframework.data.redis.connection.lettuce.LettuceReactiveGeoCommandsTests" time="0">
    <error message="Unable to connect to 127.0.0.1:6379" type="io.lettuce.core.RedisConnectionException">io.lettuce.core.RedisConnectionException: Unable to connect to 127.0.0.1:6379
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: syscall:getsockopt(..) failed: Connection refused: /127.0.0.1:6379
Caused by: io.netty.channel.unix.Errors$NativeConnectException: syscall:getsockopt(..) failed: Connection refused
</error>
  </testcase>
  <testcase name="geoHashNotExisting[org.springframework.data.redis.connection.lettuce.StandaloneConnectionProvider@18c8a027]" classname="org.springframework.data.redis.connection.lettuce.LettuceReactiveGeoCommandsTests" time="0">
    <error message="Unable to connect to 127.0.0.1:6379" type="io.lettuce.core.RedisConnectionException">io.lettuce.core.RedisConnectionException: Unable to connect to 127.0.0.1:6379
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: syscall:getsockopt(..) failed: Connection refused: /127.0.0.1:6379
Caused by: io.netty.channel.unix.Errors$NativeConnectException: syscall:getsockopt(..) failed: Connection refused
</error>
  </testcase>
  <testcase name="geoRadiusByMemberShouldApplyLimit[org.springframework.data.redis.connection.lettuce.StandaloneConnectionProvider@18c8a027]" classname="org.springframework.data.redis.connection.lettuce.LettuceReactiveGeoCommandsTests" time="0">
    <error message="Unable to connect to 127.0.0.1:6379" type="io.lettuce.core.RedisConnectionException">io.lettuce.core.RedisConnectionException: Unable to connect to 127.0.0.1:6379
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: syscall:getsockopt(..) failed: Connection refused: /127.0.0.1:6379
Caused by: io.netty.channel.unix.Errors$NativeConnectException: syscall:getsockopt(..) failed: Connection refused
</error>
  </testcase>
  <testcase name="geoRadiusByMemberShouldApplyLimit[org.springframework.data.redis.connection.lettuce.StandaloneConnectionProvider@18c8a027]" classname="org.springframework.data.redis.connection.lettuce.LettuceReactiveGeoCommandsTests" time="0">
    <error message="Unable to connect to 127.0.0.1:6379" type="io.lettuce.core.RedisConnectionException">io.lettuce.core.RedisConnectionException: Unable to connect to 127.0.0.1:6379
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: syscall:getsockopt(..) failed: Connection refused: /127.0.0.1:6379
Caused by: io.netty.channel.unix.Errors$NativeConnectException: syscall:getsockopt(..) failed: Connection refused
</error>
  </testcase>
  <testcase name="geoHash[org.springframework.data.redis.connection.lettuce.StandaloneConnectionProvider@18c8a027]" classname="org.springframework.data.redis.connection.lettuce.LettuceReactiveGeoCommandsTests" time="0">
    <error message="Unable to connect to 127.0.0.1:6379" type="io.lettuce.core.RedisConnectionException">io.lettuce.core.RedisConnectionException: Unable to connect to 127.0.0.1:6379
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: syscall:getsockopt(..) failed: Connection refused: /127.0.0.1:6379
Caused by: io.netty.channel.unix.Errors$NativeConnectException: syscall:getsockopt(..) failed: Connection refused
</error>
  </testcase>
  <testcase name="geoHash[org.springframework.data.redis.connection.lettuce.StandaloneConnectionProvider@18c8a027]" classname="org.springframework.data.redis.connection.lettuce.LettuceReactiveGeoCommandsTests" time="0">
    <error message="Unable to connect to 127.0.0.1:6379" type="io.lettuce.core.RedisConnectionException">io.lettuce.core.RedisConnectionException: Unable to connect to 127.0.0.1:6379
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: syscall:getsockopt(..) failed: Connection refused: /127.0.0.1:6379
Caused by: io.netty.channel.unix.Errors$NativeConnectException: syscall:getsockopt(..) failed: Connection refused
</error>
  </testcase>
  <testcase name="geoRadiusShouldApplyLimit[org.springframework.data.redis.connection.lettuce.StandaloneConnectionProvider@18c8a027]" classname="org.springframework.data.redis.connection.lettuce.LettuceReactiveGeoCommandsTests" time="0">
    <error message="Unable to connect to 127.0.0.1:6379" type="io.lettuce.core.RedisConnectionException">io.lettuce.core.RedisConnectionException: Unable to connect to 127.0.0.1:6379
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: syscall:getsockopt(..) failed: Connection refused: /127.0.0.1:6379
Caused by: io.netty.channel.unix.Errors$NativeConnectException: syscall:getsockopt(..) failed: Connection refused
</error>
  </testcase>
  <testcase name="geoRadiusShouldApplyLimit[org.springframework.data.redis.connection.lettuce.StandaloneConnectionProvider@18c8a027]" classname="org.springframework.data.redis.connection.lettuce.LettuceReactiveGeoCommandsTests" time="0">
    <error message="Unable to connect to 127.0.0.1:6379" type="io.lettuce.core.RedisConnectionException">io.lettuce.core.RedisConnectionException: Unable to connect to 127.0.0.1:6379
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: syscall:getsockopt(..) failed: Connection refused: /127.0.0.1:6379
Caused by: io.netty.channel.unix.Errors$NativeConnectException: syscall:getsockopt(..) failed: Connection refused
</error>
  </testcase>
  <testcase name="geoRadiusByMemberShouldReturnDistanceCorrectly[org.springframework.data.redis.connection.lettuce.StandaloneConnectionProvider@18c8a027]" classname="org.springframework.data.redis.connection.lettuce.LettuceReactiveGeoCommandsTests" time="0">
    <error message="Unable to connect to 127.0.0.1:6379" type="io.lettuce.core.RedisConnectionException">io.lettuce.core.RedisConnectionException: Unable to connect to 127.0.0.1:6379
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: syscall:getsockopt(..) failed: Connection refused: /127.0.0.1:6379
Caused by: io.netty.channel.unix.Errors$NativeConnectException: syscall:getsockopt(..) failed: Connection refused
</error>
  </testcase>
  <testcase name="geoRadiusByMemberShouldReturnDistanceCorrectly[org.springframework.data.redis.connection.lettuce.StandaloneConnectionProvider@18c8a027]" classname="org.springframework.data.redis.connection.lettuce.LettuceReactiveGeoCommandsTests" time="0">
    <error message="Unable to connect to 127.0.0.1:6379" type="io.lettuce.core.RedisConnectionException">io.lettuce.core.RedisConnectionException: Unable to connect to 127.0.0.1:6379
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: syscall:getsockopt(..) failed: Connection refused: /127.0.0.1:6379
Caused by: io.netty.channel.unix.Errors$NativeConnectException: syscall:getsockopt(..) failed: Connection refused
</error>
  </testcase>
  <testcase name="geoAddShouldAddSingleGeoLocationCorrectly[org.springframework.data.redis.connection.lettuce.StandaloneConnectionProvider@18c8a027]" classname="org.springframework.data.redis.connection.lettuce.LettuceReactiveGeoCommandsTests" time="0">
    <error message="Unable to connect to 127.0.0.1:6379" type="io.lettuce.core.RedisConnectionException">io.lettuce.core.RedisConnectionException: Unable to connect to 127.0.0.1:6379
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: syscall:getsockopt(..) failed: Connection refused: /127.0.0.1:6379
Caused by: io.netty.channel.unix.Errors$NativeConnectException: syscall:getsockopt(..) failed: Connection refused
</error>
  </testcase>
  <testcase name="geoAddShouldAddSingleGeoLocationCorrectly[org.springframework.data.redis.connection.lettuce.StandaloneConnectionProvider@18c8a027]" classname="org.springframework.data.redis.connection.lettuce.LettuceReactiveGeoCommandsTests" time="0">
    <error message="Unable to connect to 127.0.0.1:6379" type="io.lettuce.core.RedisConnectionException">io.lettuce.core.RedisConnectionException: Unable to connect to 127.0.0.1:6379
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: syscall:getsockopt(..) failed: Connection refused: /127.0.0.1:6379
Caused by: io.netty.channel.unix.Errors$NativeConnectException: syscall:getsockopt(..) failed: Connection refused
</error>
  </testcase>
  <testcase name="geoRadiusByMemberShouldReturnMembersCorrectly[org.springframework.data.redis.connection.lettuce.StandaloneConnectionProvider@18c8a027]" classname="org.springframework.data.redis.connection.lettuce.LettuceReactiveGeoCommandsTests" time="0">
    <error message="Unable to connect to 127.0.0.1:6379" type="io.lettuce.core.RedisConnectionException">io.lettuce.core.RedisConnectionException: Unable to connect to 127.0.0.1:6379
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: syscall:getsockopt(..) failed: Connection refused: /127.0.0.1:6379
Caused by: io.netty.channel.unix.Errors$NativeConnectException: syscall:getsockopt(..) failed: Connection refused
</error>
  </testcase>
  <testcase name="geoRadiusByMemberShouldReturnMembersCorrectly[org.springframework.data.redis.connection.lettuce.StandaloneConnectionProvider@18c8a027]" classname="org.springframework.data.redis.connection.lettuce.LettuceReactiveGeoCommandsTests" time="0">
    <error message="Unable to connect to 127.0.0.1:6379" type="io.lettuce.core.RedisConnectionException">io.lettuce.core.RedisConnectionException: Unable to connect to 127.0.0.1:6379
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: syscall:getsockopt(..) failed: Connection refused: /127.0.0.1:6379
Caused by: io.netty.channel.unix.Errors$NativeConnectException: syscall:getsockopt(..) failed: Connection refused
</error>
  </testcase>
  <testcase name="geoRadiusShouldReturnDistanceCorrectly[org.springframework.data.redis.connection.lettuce.StandaloneConnectionProvider@18c8a027]" classname="org.springframework.data.redis.connection.lettuce.LettuceReactiveGeoCommandsTests" time="0">
    <error message="Unable to connect to 127.0.0.1:6379" type="io.lettuce.core.RedisConnectionException">io.lettuce.core.RedisConnectionException: Unable to connect to 127.0.0.1:6379
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: syscall:getsockopt(..) failed: Connection refused: /127.0.0.1:6379
Caused by: io.netty.channel.unix.Errors$NativeConnectException: syscall:getsockopt(..) failed: Connection refused
</error>
  </testcase>
  <testcase name="geoRadiusShouldReturnDistanceCorrectly[org.springframework.data.redis.connection.lettuce.StandaloneConnectionProvider@18c8a027]" classname="org.springframework.data.redis.connection.lettuce.LettuceReactiveGeoCommandsTests" time="0">
    <error message="Unable to connect to 127.0.0.1:6379" type="io.lettuce.core.RedisConnectionException">io.lettuce.core.RedisConnectionException: Unable to connect to 127.0.0.1:6379
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: syscall:getsockopt(..) failed: Connection refused: /127.0.0.1:6379
Caused by: io.netty.channel.unix.Errors$NativeConnectException: syscall:getsockopt(..) failed: Connection refused
</error>
  </testcase>
  <testcase name="geoPosNonExisting[org.springframework.data.redis.connection.lettuce.StandaloneConnectionProvider@18c8a027]" classname="org.springframework.data.redis.connection.lettuce.LettuceReactiveGeoCommandsTests" time="0">
    <error message="Unable to connect to 127.0.0.1:6379" type="io.lettuce.core.RedisConnectionException">io.lettuce.core.RedisConnectionException: Unable to connect to 127.0.0.1:6379
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: syscall:getsockopt(..) failed: Connection refused: /127.0.0.1:6379
Caused by: io.netty.channel.unix.Errors$NativeConnectException: syscall:getsockopt(..) failed: Connection refused
</error>
  </testcase>
  <testcase name="geoPosNonExisting[org.springframework.data.redis.connection.lettuce.StandaloneConnectionProvider@18c8a027]" classname="org.springframework.data.redis.connection.lettuce.LettuceReactiveGeoCommandsTests" time="0">
    <error message="Unable to connect to 127.0.0.1:6379" type="io.lettuce.core.RedisConnectionException">io.lettuce.core.RedisConnectionException: Unable to connect to 127.0.0.1:6379
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: syscall:getsockopt(..) failed: Connection refused: /127.0.0.1:6379
Caused by: io.netty.channel.unix.Errors$NativeConnectException: syscall:getsockopt(..) failed: Connection refused
</error>
  </testcase>
  <testcase name="geoRadiusShouldReturnMembersCorrectly[org.springframework.data.redis.connection.lettuce.StandaloneConnectionProvider@18c8a027]" classname="org.springframework.data.redis.connection.lettuce.LettuceReactiveGeoCommandsTests" time="0">
    <error message="Unable to connect to 127.0.0.1:6379" type="io.lettuce.core.RedisConnectionException">io.lettuce.core.RedisConnectionException: Unable to connect to 127.0.0.1:6379
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: syscall:getsockopt(..) failed: Connection refused: /127.0.0.1:6379
Caused by: io.netty.channel.unix.Errors$NativeConnectException: syscall:getsockopt(..) failed: Connection refused
</error>
  </testcase>
  <testcase name="geoRadiusShouldReturnMembersCorrectly[org.springframework.data.redis.connection.lettuce.StandaloneConnectionProvider@18c8a027]" classname="org.springframework.data.redis.connection.lettuce.LettuceReactiveGeoCommandsTests" time="0">
    <error message="Unable to connect to 127.0.0.1:6379" type="io.lettuce.core.RedisConnectionException">io.lettuce.core.RedisConnectionException: Unable to connect to 127.0.0.1:6379
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: syscall:getsockopt(..) failed: Connection refused: /127.0.0.1:6379
Caused by: io.netty.channel.unix.Errors$NativeConnectException: syscall:getsockopt(..) failed: Connection refused
</error>
  </testcase>
  <testcase name="geoDistShouldReturnDistanceInMetersByDefault[org.springframework.data.redis.connection.lettuce.StandaloneConnectionProvider@18c8a027]" classname="org.springframework.data.redis.connection.lettuce.LettuceReactiveGeoCommandsTests" time="0">
    <error message="Unable to connect to 127.0.0.1:6379" type="io.lettuce.core.RedisConnectionException">io.lettuce.core.RedisConnectionException: Unable to connect to 127.0.0.1:6379
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: syscall:getsockopt(..) failed: Connection refused: /127.0.0.1:6379
Caused by: io.netty.channel.unix.Errors$NativeConnectException: syscall:getsockopt(..) failed: Connection refused
</error>
  </testcase>
  <testcase name="geoDistShouldReturnDistanceInMetersByDefault[org.springframework.data.redis.connection.lettuce.StandaloneConnectionProvider@18c8a027]" classname="org.springframework.data.redis.connection.lettuce.LettuceReactiveGeoCommandsTests" time="0">
    <error message="Unable to connect to 127.0.0.1:6379" type="io.lettuce.core.RedisConnectionException">io.lettuce.core.RedisConnectionException: Unable to connect to 127.0.0.1:6379
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: syscall:getsockopt(..) failed: Connection refused: /127.0.0.1:6379
Caused by: io.netty.channel.unix.Errors$NativeConnectException: syscall:getsockopt(..) failed: Connection refused
</error>
  </testcase>
</testsuite>